setup.py

```diff
@@ -1,58 +0,0 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-import sys
-import codecs
-from setuptools import setup, find_packages
-
-
-setup(
-    name='sphinxcontrib-images',
-    version='0.9.4',
-    url='https://github.com/sphinx-contrib/images',
-    download_url='https://pypi.python.org/pypi/sphinxcontrib-images',
-    project_urls={
-        'Bug Tracker': 'https://github.com/sphinx-contrib/images/issues',
-        'Documentation': 'https://sphinxcontrib-images.readthedocs.io/',
-    },
-    license='Apache 2',
-    author=u'Tomasz Czyż',
-    author_email='tomasz.czyz@gmail.com',
-    description='Sphinx extension for thumbnails',
-    long_description=codecs.open('README.rst', encoding="utf8").read(),
-    zip_safe=False,
```

Comment:
This looks like it's deprecated and doesn't seem to apply to Python 3 (or it can still break since you _can_ install zip packages, but that's not standard and)